package.json

```diff
@@ -11,7 +11,10 @@
 	},
 	"devDependencies": {
 		"@types/node": "^16.0.0",
-		"typescript": "4.x",
-		"ttypescript": "1.x"
+		"ttypescript": "1.x",
+		"typescript": "4.x"
+	},
+	"dependencies": {
+		"ts-node": "^10.4.0"
```

Comment:
`ts-node` should not be dependency. This wasn't intended, right?

Reply:
I'll change it to a dev a dep - can easily run and debug from the ide with ts-node, for some reason my global installation isn't working :(